src/TypeScript.UnitTests/TsConfig/TsConfigProviderTests.cs

```diff
@@ -135,39 +135,36 @@ public async Task GetConfigForFile_TsConfigContainsFileWithIllegalCharacters_Fil
 
             var testSubject = CreateTestSubject(tsConfigsLocator.Object, eslintBridgeClient.Object);
             var result = await testSubject.GetConfigForFile(testedFileName, CancellationToken.None);
-            result.Should().Be("config");
-
-            eslintBridgeClient.Verify(x => x.TsConfigFiles("config", CancellationToken.None), Times.Once);
```

Comment:
Unrelated: this assertion was true but irrelevant so I removed it to make the test clearer what it was testing.